packages/app-project/stores/YourStats.js

```diff
@@ -0,0 +1,91 @@
+import { autorun } from 'mobx'
+import { addDisposer, flow, getRoot, types } from 'mobx-state-tree'
+import auth from 'panoptes-client/lib/auth'
+import asyncStates from '@zooniverse/async-states'
+import { panoptes } from '@zooniverse/panoptes-js'
+
+const YourStats = types
+  .model('YourStats', {
+    dailyCounts: types.array(types.frozen({})),
+    error: types.maybeNull(types.frozen({})),
+    loadingState: types.optional(types.enumeration('state', asyncStates.values), asyncStates.initialized),
+    totalCount: types.optional(types.number, 0)
+  })
+
+  .views(self => ({
+    get todaysCount () {
+      return self.dailyCounts[0]
+    }
+  }))
+
+  .actions(self => {
+    function createProjectObserver () {
+      const projectDisposer = autorun(() => {
+        const { project, user } = getRoot(self)
+        if (project.id && user.id) {
+          self.fetchActivityCount()
+          self.fetchDailyCounts()
+        }
+      })
+      addDisposer(self, projectDisposer)
+    }
+
+    return {
+      afterAttach () {
+        createProjectObserver()
+      },
+
+      fetchActivityCount: flow(function * fetchActivityCount () {
+        const { project, user } = getRoot(self)
+        self.loadingState = asyncStates.loading
+        try {
+          const token = yield auth.checkBearerToken()
+          const authorization = `Bearer ${token}`
+          const query = {
+            project_id: project.id,
+            user_id: user.id
+          }
+          // TODO: this should really share the UPP that's being requested by the classifier.
+          const response = yield panoptes.get('/project_preferences', query, authorization)
+          const [ preferences ] = response.body.project_preferences
+          self.totalCount = preferences ? preferences.activity_count : 0
+        } catch (error) {
+          console.log(error.message)
+          self.error = error
+          self.loadingState = asyncStates.error
+        }
+      }),
+
+      fetchDailyCounts: flow(function * fetchDailyCounts () {
+        const { project, user } = getRoot(self)
+        self.loadingState = asyncStates.loading
+        try {
+          const token = yield auth.checkBearerToken()
+          const host = 'https://stats.zooniverse.org'
+          const endpoint = '/graphql'
+          const authorization = `Bearer ${token}`
+          const query = {
```

Comment:
I need to get the igraphql interface working on staging / production to test queries against. It's on my list todo and i'll also document the API once i figure it out. However we can use the specs to get a headstart on the graphql payloads. 

Based on the specs here, https://github.com/zooniverse/zoo-stats-api-graphql/blob/6673e3a7e85b8938f80aa1f38b394f40e85c1313/spec/graphql/zoo_stats_schema_spec.rb#L172

and how the controller fetches param payloads here https://github.com/zooniverse/zoo-stats-api-graphql/blob/0c971d5e9d415ad8fdb1e5b240c9e05ab831d50e/app/controllers/graphql_controller.rb#L8

You may need `operation_name` param as well, https://graphql.org/learn/queries/#operation-name

I think the payload needs be correctly keyed strings / json data (not sure how this gets represented correctly yet). 
```
{
	"query": "query ($userId: ID!, $eventType: String!, $interval: String!){ statsCount(userId: $userId, eventType: $eventType, interval: $interval) { period, count }",
	"variables": "{ userId: current_user, eventType: 'classification', interval: '1 day'}"
}
```

It looks like scribes pulls data from caesar using https://www.npmjs.com/package/graphql-request so it might be worth looking into that lib for details on how they package up the POST payloads,  

https://github.com/prisma/graphql-request/blob/b0abe80788741f6360c7cd2cbc37620c573be0b1/src/index.ts#L20

Reply:
Thanks! I'll have a look. I've been using the examples from https://graphql.org/ to learn how GraphQL queries work.

Reply:
I'm going to add the `graphql-request` package and give that a go.